tests/rptest/tests/partition_movement_test.py

```diff
@@ -293,11 +295,13 @@ def test_move_consumer_offsets_intranode(self):
     @cluster(num_nodes=5,
              log_allow_list=PARTITION_MOVEMENT_LOG_ERRORS +
              RESTART_LOG_ALLOW_LIST)
-    def test_bootstrapping_after_move(self):
+    @parametrize(test_upgrade=True)
+    @parametrize(test_upgrade=False)
+    def test_bootstrapping_after_move(self, test_upgrade):
         """
         Move partitions with active consumer / producer
         """
-        self.start_redpanda(num_nodes=3)
+        self.start_redpanda(num_nodes=3, install_previous_version=test_upgrade)
```

Comment:
I see this installing a different redpanda depending on `test_upgrade`, but I don't see a different version getting installed later in the test -- is there no actual upgrade happening in this test?